src/constants/addresses.ts

```diff
@@ -28,5 +54,18 @@ export const addresses: ContractAddressMap = {
     MentoToken: '0xD2f4f160BAF7D88a7A9189b03D0B3AA6A5D9775B',
     TimelockController: '0xb5977b1d208ef35FAf97A9534Dd849c356F362C5',
     Locking: '0x831DAfC0912e1c2aBa2Da90668c0856d48a8C06b',
+    Broker: '0xB881aF21C5A9ff8e8d5E4C900F67F066C6CB7936',
```

Comment:
This is actually the BreakerBoxPoxy
https://explorer.celo.org/baklava/address/0xB881aF21C5A9ff8e8d5E4C900F67F066C6CB7936

This is the Broker address:
https://explorer.celo.org/baklava/address/0x6723749339e320E1EFcd9f1B0D997ecb45587208